.github/workflows/open-pr.yml

```diff
@@ -69,7 +69,7 @@ jobs:
       - name: Setup GitHub Token
         if: '!steps.check.outputs.skip'
         id: token
-        uses: smartcontractkit/.github/actions/setup-github-token@7ac9af09dda8c553593d2153a975b43b6958fa9f # setup-github-token@0.1.2
+        uses: smartcontractkit/.github/actions/setup-github-token@56eb9fe6d4dfc7a26cd666d8d6d0173c79173b98 # setup-github-token@0.2.1
```

Comment:
the sha hash should be this one:
https://github.com/smartcontractkit/.github/commit/ef78fa97bf3c77de6563db1175422703e9e6674f

Reply:
It should be fixed.